RoslynPluginGenerator/CommandLine/ProcessedArgs.cs

```diff
@@ -13,8 +13,9 @@ public class ProcessedArgs
         private readonly NuGetReference analyzerRef;
         private readonly string sqaleFilePath;
         private readonly string language;
+        private readonly bool acceptLicenses;
 
-        public ProcessedArgs(NuGetReference analyzerRef, string sqaleFilePath, string language)
+        public ProcessedArgs(NuGetReference analyzerRef, string sqaleFilePath, string language, bool acceptLicenses)
```

Comment:
That's a very general question more suited to a discussion than a PR. Are there any specific issues you want raised about this PR, or are you saying that three parameters is already too many?